package/Brewfile.cask

```diff
@@ -10,5 +10,6 @@ cask 'mendeley'
 cask 'slack'
 
 cask 'microsoft-office'
+cask 'visual-studio-code'
```

Comment:
It seems that we cannot install `Visual Studio Code - Insiders`  using homebrew.
But we can install `Visual Studio Code`. 🎉